js/modules/k6/http/ntlm_test.go

```diff
@@ -0,0 +1,171 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2018 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package http
+
+import (
+	"encoding/base64"
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/ThomsonReutersEikon/go-ntlm/ntlm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var challenges map[string]*ntlm.ChallengeMessage
+
+func TestGetCredentialsFromHeader(t *testing.T) {
+	user, pass, err := getCredentialsFromHeader("Basic Ym9iOnBhc3M=")
```

Comment:
Not sure if it's worth it to tests broken`Authorization` headers. Theoretically this could come from our users directly, not from the Go standard library converting URL credentials to a header